documentation/api/io.strimzi.api.kafka.model.KafkaMirrorMakerProducerSpec.adoc

```diff
@@ -1,18 +1,50 @@
-Use the three allowed `ssl` configuration options to run external listeners with a specific _cipher suite_ for a TLS version.
-A _cipher suite_ combines algorithms for secure connection and data transfer.
-
-.Example SSL configuration
-[source,yaml,subs="attributes+"]
-----
-spec:
-  producer:
-    config:
-      ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" <1>
-      ssl.enabled.protocols: "TLSv1.2" <2>
-      ssl.protocol: "TLSv1.2" <3>
-----
-<1> The cipher suite for TLS using a combination of `ECDHE` key exchange mechanism, `RSA` authentication algorithm,
-`AES` bulk encyption algorithm and `SHA384` MAC algorithm.
-<2> The SSl protocol `TLSv1.2` is enabled.
-<3> Specifies the `TLSv1.2` protocol to generate the SSL context.
-Allowed values are `TLSv1.1` and `TLSv1.2`.
+Use `KafkaMirrorMakerProducerSpec` properties to configure a MirrorMaker producer.
+
+[id='property-producer-abort-on-send-{context}']
+.`abortOnSendFailure`
+
+Use the `producer.abortOnSendFailure` property to configure how to handle message send failure from the producer.
+
+By default, if an error occurs when sending a message from Kafka MirrorMaker to a Kafka cluster:
+
+* The Kafka MirrorMaker container is terminated in Kubernetes.
+* The container is then recreated.
+
+If the `abortOnSendFailure` option is set to `false`, message sending errors are ignored.
+
+[id='property-producer-config-{context}']
+.`config`
+
+Use the `producer.config` properties to configure Kafka options for the producer.
+
+The `config` property contains the Kafka MirrorMaker producer configuration options as keys, with values set in one of the following JSON types:
+
+* String
+* Number
+* Boolean
+
+To run external listeners with a specific _cipher suite_ for a TLS version, you can xref:con-common-configuration-cipher-reference[configure allowed `ssl` properties].
+
+*Exceptions*
+
+You can specify and configure the options listed in the {ApacheKafkaProducerConfig}.
+
+However, there are exceptions for options automatically configured and managed directly by Strimzi related to:
+
+* Kafka cluster bootstrap address
+* Security (encryption, authentication, and authorization)
+
+Specifically, all configuration options with keys equal to or starting with one of the following strings are forbidden:
+
+* `ssl.` (with the exception of xref:con-common-configuration-cipher-reference[cipher suites])
+* `sasl.`
+* `security.`
+* `bootstrap.servers`
+* `group.id`
```

Comment:
We should add `interceptor.classes` to the list and remove `group.id` which is consumer only.